docs/src/pages/index.tsx

```diff
@@ -183,33 +266,40 @@ function Features() {
   return (
     <div className="container margin-top--lg margin-bottom--md">
       <Heading as="h1" className="margin-bottom--md text--center text--primary">
-        Features
+        Capabilities
       </Heading>
       <p className={clsx('text--center', styles.sectionSubheading)}>
         Simple, powerful, fast. Pick three.
       </p>
 
       <Feature
-        text="WebGPU APIs"
-        secondaryText="Utilize existing WebGPU-based components or craft your own WGSL shader to achieve any desired effect and directly leverage GPU parallel processing capabilities."
+        text="Built-in components"
+        secondaryText="Use built-in components to process streams, render text, images, GIFs or websites, and combine them into output streams."
+        image={<FaPuzzlePiece className={styles.featureIcon} style={{ fontSize: 80 }} />}
+      />
+
+      <Feature
+        text="Highly customizable"
```

Comment:
WebGPU logo does not fit here with this title. Either change logo or title. I personally would prefer to keep this point specific about WebGPU